scripts/start-op-nethermind.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -e
+
+# Add debug information
+echo "Debug: Checking Nethermind executable..."
+ls -l /nethermind/Nethermind.Runner || echo "Nethermind.Runner not found in /nethermind/"
```

Comment:
`Nethermind.Runner` has been replaced with `nethermind` and should be avoided:
```suggestion
ls -l /nethermind/nethermind || echo "nethermind not found in /nethermind/"
```